scss/prototype/_display.scss

```diff
@@ -0,0 +1,50 @@
+// Foundation for Sites by ZURB
+// foundation.zurb.com
+// Licensed under MIT Open Source
+
+////
+/// @group prototype-display
+////
+
+/// Responsive breakpoints for display, defaulted to global-prototype-responsive-breakpoints.
+/// @type Boolean
+$prototype-display-responsive-breakpoints: $global-prototype-responsive-breakpoints !default;
+
+@mixin foundation-prototype-display {
+  @each $display in (none, inline, inline-block, block, table, table-cell) {
```

Comment:
Thanks @IllusionPerdu ..... but will wait for **brett or nate or kevin**'s answer for it .....
as i don't wanna do rework and all!